### PR TITLE
Use object

### PR DESCRIPTION
**Pull request by Abe1able**
### In this pull request, I have

- set up the to-do list project,
- used webpack for package management,
- created an object of list items and insert them dynamically into the webpage
- used HTML, CSS, and JavaScript best practices.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  {
+    description: 'Work out',
+    completed: false,
+    index: 3,
+  },
+];
+
+const forms = document.querySelector('.todo-form');
+
+List.forEach((item) => {
```
Comment (SajjadAhmad14) [inline src/index.js:30]: The tasks should appear in order of the `index` values for each task. So if I add a task `  {
    description: 'test Project',
    completed: true,
    index: 0,
  },`
this should be displayed as the first item on the list.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Abel, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (Abe1able) [reply src/index.js:30]: Good challenge. Done.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Abe1able, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You did well to make the changes requested by the previous reviewer✔️✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**